src/routes/v2/authenticated/review.ts

```diff
@@ -924,19 +979,24 @@ export class ReviewsRouter {
   > = async (req, res) => {
     // Step 1: Check that the site exists
     const { siteName, requestId } = req.params
+    // Check that user exists.
+    // Having session data is proof that this user exists
+    // as otherwise, they would be rejected by our middleware
+    const { userWithSiteSessionData } = res.locals
     const site = await this.sitesService.getBySiteName(siteName)
+    const doesUnmigratedSiteExist = await this.doesUnmigratedSiteExist(
+      userWithSiteSessionData
+    )
+
+    if (!site && doesUnmigratedSiteExist)
+      return res.status(200).json({ message: "Unmigrated site" })
```

Comment:
I think we might not need the check here, since this is a POST?

Reply:
removed